stubs/AMQP.php

```diff
@@ -66,11 +66,16 @@
 
 /**
  * When publishing a message, the message must be routed to a valid queue. If it is not, an error will be returned.
+ * Note: This flag does not currently work in pecl-amqp, and WILL cause undefined behavior if you use it.
+ * @link https://bugs.php.net/bug.php?id=62786
+ * @link https://github.com/pdezwart/php-amqp/issues/23
  */
 define('AMQP_MANDATORY', 1024);
 
 /**
- * When publishing a message, mark this message for immediate processing by the broker. (High priority message.)
+ * When publishing a message, the message must be routed to a valid queue with an available consumer. If it is not, an
```

Comment:
Yes, I know - I read the AMQP spec when writing that comment. IMO 'mark this message for immediate processing by the broker' doesn't really communicate the point that this message will fail only if no consumer is available to handle it immediately on that queue. I'm aware that RabbitMQ does not implement the immediate flag, but php-amqp will fail with an exception telling you so (which is why I didn't add any disclaimers on that flag).